src/createKindeClient.ts

```
@@ -393,49 +420,46 @@ const createKindeClient = async (
 
-    if (audience) {
-      /* if multiple audiences requested it should appear multiple times in the query string */
-      audience
-        .trim()
-        .split(/\s+/)
-        .forEach((aud) => {
-          urlSearchParams.append('audience', aud);
-        });
+    if (Object.keys(authUrlParams).length > 0) {
+      const merged = new URLSearchParams(url.search);
+      Object.entries(authUrlParams as Record<string, string>).forEach(
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)